src/process-analysis/planning.tex

```diff
@@ -1,5 +1,40 @@
 % textidote: ignore begin
-
-
 \section{Planning}\label{sec:planning}
 % textidote: ignore end
+
+The authors of this project have been working together since P0 and have
+therefore developed some mixed habits when working together.
+The main issue when working on the projects has been bad oversight
+into how much work goes into specific parts of developing a product.
+In the P1 project~\cite{P1}, the main problem was lack of time for improving the code
+and implementing fixes that were found in the wake of the release.
+
+For the P2 project, the group decided to introduce scrum as a tool to
+better plan work, as we saw a need for it to combat pushing most of the
+work too close to the deadline.
+
+Scrum is a framework for working in an agile environment~\cite{agile-manifesto}.
+To fully implement Scrum would be overkill for this project, and the authors
+therefore decided to cut back on some of the more stringent requirements of Scrum.
+The implementation used during the P2 project involved weekly virtual or physical meetings where each team member
+presented what they had been working on.
+This also functioned as a good place to ask for assistance or other questions the members had.
+
+To get a better overview of all the work that should be done, we made use of GitHub Issues~\cite{github-issues}.
+This allows us to integrate GitHub as our version control to automate some management of tasks.
+As an example of GitHub Issues usefulness, when completing a task through a pull request using a keyword in the commit
+message would make GitHub move the task from doing to done.
+One could have used other task trackers such as Trello, but GitHub Issues had many Quality of Life improvements because
+it is integrated in GitHub itself and was therefore the preferred issue tracker/task manager.
```

Comment:
I would say that this describes a tool we used, which is a main report segment that already exists. The process analysis is more about reflecting on the process rather than the benefits of the tools. Consider rewriting this segment.